packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterBuilder.tsx

```diff
@@ -69,7 +73,25 @@ export function PresentationInstanceFilterBuilder(props: PresentationInstanceFil
         initialFilter={initialPropertyFilter}
         onFilterChanged={onFilterChanged}
         ruleGroupDepthLimit={ruleGroupDepthLimit}
+        ruleValueRenderer={
+          enableUniqueValuesRenderer
+            ? (pr: PropertyFilterBuilderRuleValueProps) => <UniqueValuesRenderer {...pr} imodel={imodel} descriptor={descriptor} />
+            : undefined
+        }
       />
     </navigationPropertyEditorContext.Provider>
   );
 }
+
+function UniqueValuesRenderer(props: PropertyFilterBuilderRuleValueProps & { imodel: IModelConnection; descriptor: Descriptor }) {
+  return (
+    <>
+      {props.property.typename !== StandardTypeNames.Navigation &&
+      (props.operator === PropertyFilterRuleOperator.IsEqual || props.operator === PropertyFilterRuleOperator.IsNotEqual) ? (
```

Comment:
Looks like this can be refactored to `if` statement instead of ternary operator.

Reply:
fixed